chia/types/eligible_coin_spends.py

```diff
@@ -233,26 +233,30 @@ async def process_fast_forward_spends(
         get_unspent_lineage_info_for_puzzle_hash: Callable[[bytes32], Awaitable[Optional[UnspentLineageInfo]]],
         height: uint32,
         constants: ConsensusConstants,
-    ) -> None:
+    ) -> InternalMempoolItem:
         """
-        Provides the caller with an in-place internal mempool item that has a
-        proper state of fast forwarded coin spends and additions starting from
+        Provides the caller with an internal mempool item that has a proper
+        state of fast forwarded coin spends and additions starting from
         the most recent unspent versions of the related singleton spends.
 
         Args:
-            mempool_item: in-out parameter for the internal mempool item to process
+            mempool_item: The internal mempool item to process
             get_unspent_lineage_info_for_puzzle_hash: to lookup the most recent
                 version of the singleton from the coin store
             constants: needed in order to refresh the mempool item if needed
             height: needed in order to refresh the mempool item if needed
 
+        Returns:
+            The resulting mempool item after fast forwarding
+
         Raises:
             If a fast forward cannot proceed, to prevent potential double spends
         """
+        resulting_item = dataclasses.replace(mempool_item)
```

Comment:
is this just creating a copy? `replace()` typically also  take arguments of the fields to replace in the copy. it seems confusing to use `replace()` to just mean `copy`.

Reply:
Yes. I added a comment to clarify this.